spring-petclinic-microservices/spring-petclinic-visits-service/pom.xml

```diff
@@ -7,6 +7,7 @@
 	<artifactId>spring-petclinic-visits-service</artifactId>
 	<packaging>jar</packaging>
 	<description>Spring PetClinic Visits Service</description>
+    <name>Spring PetClinic Visits Service</name>
```

Comment:
indent problem.

Reply:
the original petclinic samples don't have the same patten, the issue can be tracked in this [issue](https://github.com/Azure/azure-sdk-for-java/issues/26234).

![image](https://user-images.githubusercontent.com/4465723/148526266-8c2b98d0-a10e-4654-9227-7f526876cef0.png)